### DISCUSSION TITLE
Is contribution open to anyone around the world?

### DISCUSSION THREAD

**Discussion by yujioshiro** [TEAMMATES/teammates#12374] [Q&A] [answered]
I am a second-year CS student in the US, have 0 relationship with TEAMMATES or NUS. I'm looking at contributing to some OSS for more real world experience, am I allowed to work on issues?

**Comment by weiquu** [comment, marked as answer]
Hi @yujioshiro, yes feel free to work on issues! You can take a look at the [Contributing page](https://teammates.github.io/teammates/contributing-doc.html) in our docs for more information. Do also read through the other pages, in particular those under Workflow.
